### DISCUSSION TITLE
What source images for building to get the best possible face resemblance

### DISCUSSION THREAD

**Discussion by a3nima** [glucauze/sd-webui-faceswaplab#16] [Q&A]
I found that some persons are very well represented. It's a lot of fun to work with these. On the other hand, on some it seems to work not so well. 

So my question: Is there a "best practice"-way of doing the face-model building? What's your experience?

How many images ? 
Different angles or not?
Lighting/shadows on Face - yes /no?
Min/Max quality? File type /size?

Thanks!

**Comment by glucauze** [comment]
It's true that it depends a lot on the faces. I don't really have any advice on that. 

+ Sometimes using several faces helps to average the face.
+ Sometimes going over the front face with codeformer can give a better result.

It varies a lot. Maybe others will have ideas on the subject.
